internal/builtinfunctions/functions.go

```diff
@@ -61,7 +62,7 @@ func getIntToFloatFunction() schema.CallableFunction {
 		false,
 		schema.NewDisplayValue(
 			schema.PointerTo("intToFloat"),
-			schema.PointerTo("Converts an integer type into a floating point type."),
+			schema.PointerTo("Returns a floating-point representation of the inputted integer."),
```

Comment:
"inputted"?? Maybe just "integer parameter"?? Or in some later examples you seem to just use "input", which in context is probably fine.

Reply:
Changed.